packages/mui-icons-material/lib/InsertChartOutlined.js

```diff
@@ -12,7 +12,7 @@ var _createSvgIcon = _interopRequireDefault(require("./utils/createSvgIcon"));
 var _jsxRuntime = require("react/jsx-runtime");
 
 var _default = (0, _createSvgIcon.default)( /*#__PURE__*/(0, _jsxRuntime.jsx)("path", {
-  d: "M9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4zm2.5 2.1h-15V5h15v14.1zm0-16.1h-15c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h15c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
+  d: "M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 16H5V5h14v14zM7 10h2v7H7zm4-3h2v10h-2zm4 6h2v4h-2z"
```

Comment:
Why are the icons changed? If there was an update from Google since the last time we synced, please create a separate PR and keep this one focused on ignored icons.

Reply:
I reran the commands after consolidating the list to make sure it was no longer downloading the icons from the `blacklistedIcons` list and so it removes those downloaded files. Should I not have?

Reply:
Should I revert `change in packages/mui-icons-material/lib/esm/InsertChartOutlined.js` and `packages/mui-icons-material/lib/InsertChartOutlined.js` and keep the other deletions/changes?

Reply:
I personally didn't see an issue with keeping those changes. In the unlikely event that this PR gets reverted, they'll be picked up in a future update.